src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/ModifiedControlVariableCheck.java

```diff
@@ -92,8 +113,8 @@
  * <p>Example:</p>
  *
  * <pre>
- * for (String line: lines) {
- *   line = line.trim();   // it will skip this violation
+ * for (String arg: args) {
+ *   arg = arg.trim();   // ok
```

Comment:
Please move this forech loop to first example to show that there is no violation on such code by default.
Please use absolutely the same java code for each  example